src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -110,9 +110,12 @@ namespace
         ParamIdAdaptiveTileNoiseThreshold               = 44,
         ParamIdBackgroundAlphaValue                     = 15,
         ParamIdNoiseSeed                                = 76,
+        ParamIdEnableNoiseSeed                          = 77,
 
         ParamIdLightingAlgorithm                        = 52,
         ParamIdForceDefaultLightsOff                    = 13,
+        ParamIdLightSamplingAlgorithm                   = 78,
+        ParamIdEnableImportanceSampling                 = 79,
```

Comment:
We need to be more specific here: `ParamIdEnableLightImportanceSampling`.

Reply:
ok